test/js/length-value.js

```diff
@@ -1,15 +1,15 @@
 suite('LengthValue', function() {
   test('fromValue returns instanceof a LengthValue and StyleValue', function() {
-    var fromVal = (new LengthValue()).fromValue(9.2, 'px');
-    assert.instanceOf(fromVal, SimpleLength, 'A new fromValue should be an instance of SimpleLength');
-    assert.instanceOf(fromVal, LengthValue, 'A new fromValue should be an instance of LengthValue');
-    assert.instanceOf(fromVal, StyleValue, 'A new fromValue should be an instance of StyleValue');
+    var fromValue = (new LengthValue()).fromValue(9.2, 'px');
```

Comment:
should be fine to just call this value, here and elsewhere. "fromDictionary" sounds like it should be a dictionary, which is a bit confusing...